.github/workflows/lock-threads.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
Please add a description for this setting (cron settings are not that easy to remember).

Reply:
[crontab.guru](https://crontab.guru/#0_*_*_*_* ) is handy. :)

> “At minute 0.”